src/test/java/kg/jarkyn/server/Resource/ResourceReaderTest.java

```diff
@@ -0,0 +1,22 @@
+package kg.jarkyn.server.Resource;
+
+import kg.jarkyn.server.FileHelpers;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class ResourceReaderTest {
+
+    @Test
+    public void returnsFileByteContent() {
+        byte[] expectedContent = FileHelpers.plainFileByteContent();
+
+        assertArrayEquals(expectedContent, ResourceReader.readFile(FileHelpers.plainFileFullPath));
+    }
+
+    @Test
+    public void listsFolderContents() {
+        assertEquals(FileHelpers.publicFolderListing, ResourceReader.readFolder(FileHelpers.publicFolderPath));
```

Comment:
That's not really verifying behaviour of `ResourceReader`, is it? Only that `FileHelpers` has the correct hardcoded values.